source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5906,13 +5906,17 @@ export abstract class StylableCardElementContainer extends CardElementContainer
     }
 
     protected getHasBackground(): boolean {
+        return this.getHasPadding(true);
+    }
+
+    protected getHasPadding(shouldCountBackgroundImage : boolean): boolean {
```

Comment:
name here is very confusion.
Possible add a function getHasBackgroundImage which is checking if background image is set or not.
then getHasBackground 
```
condition && getHasBackgroundImage()
```

Possible the `getHasPadding`
```
return shouldCountBackgroundImage && getHasBackgroundImage
```

Reply:
@licanhua; I'm not quite sure if I'm following you. getHasPadding return true if this container has padding and return false if it doesn't.